packages/utils/interactive-hooks/src/useKeyProps.windows.ts

```diff
@@ -72,3 +72,22 @@ export const useKeyProps = memoize(getKeyUpPropsWorker);
 
 /** Exposes the behavior of useKeyProps for the current platform as a boolean */
 export const preferKeyDownForKeyEvents = false;
+
+/**
+ * Verifies if nativeEvent contains modifier key.
+ * @param nativeEvent
+ * @returns `true` if one or more of modifier keys are `true`
+ */
+export const isModifierKey = (nativeEvent: any): boolean => {
```

Comment:
I had to use `any` for native event parameter because this type is `Readonly<{ key: string; target?: number; eventCounter?: number; }>` but nativeEvent, also, contains modifier keys and if I use the type with modifier keys there will be an error about type mismatch

Reply:
Especially with Event types in callbacks, I've found often the TS types make like difficult / differ a lot per platform, so I've been using `any` myself. I think this is fine.